### PR TITLE
i18n(zh-cn): update `guides/troubleshooting.mdx`

### PR DESCRIPTION
**Pull request by dorasans**
#### What kind of changes does this PR include?

- update the translation

#### Description
- update `guides/troubleshooting.mdx`

### CODE REVIEW

**Comment by dorasans** [comment]
@liruifengv pls review

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 1]
```diff
@@ -133,6 +133,27 @@ Yarn 2+，又名 Berry，使用名叫 [Plug'n'Play (PnP)](https://yarnpkg.com/fe
 nodeLinker: "node-modules"
 ```
 
+### 在单一代码库中为 Astro 添加依赖。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:136]:
```suggestion
### 在单一代码库中为 Astro 添加依赖
```
标题里面可以不用加句号

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 2]
```diff
@@ -200,6 +221,34 @@ const answer = sum(2, 4);
...
 
+## 创建最小化复现
+
+当你排查代码时，创建一个 **最小化复现** 的问题可以很有帮助，你可以将其分享出来。这是一个更小、更简化的 Astro 项目，用于演示你遇到的问题。在一个新的项目中一个有效的复现有助于确认这是否是一个可重复的问题，而不是由于你个人环境或现有项目中的其他因素引起的。
+
+在我们的技术支持论坛寻求帮助时，分享一个最小化复现对于解决问题非常有帮助，并且这在向 Astro 提交错误报告时通常是必需的。
+
+### 创建一个 StackBlitz 通过 [astro.new](https://astro.new)
+
+你可以使用 [astro.new](https://astro.new) 来一键创建一个新的Astro项目。对于最小化复现，我们强烈建议从在 [StackBlitz](https://stackblitz.com) 上运行的最小（空白）示例开始，尽量少添加额外的代码。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:232]:
```suggestion
你可以使用 [astro.new](https://astro.new) 来一键创建一个新的 Astro 项目。对于最小化复现，我们强烈建议在 [StackBlitz](https://stackblitz.com) 上运行的最小（空白）示例开始，尽量少添加额外的代码。
```

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 3]
```diff
@@ -200,6 +221,34 @@ const answer = sum(2, 4);
...
+
+StackBlitz 将在浏览器中运行这个 Astro 项目，而不是在你的本地环境中。它还会为你提供一个可分享的链接，这样任何 Astro 的维护者或支持团队成员都可以在他们自己的本地环境之外查看你的最小化复现。这意味着每个人查看的都是完全相同的项目，具有相同的配置和依赖。这使得其他人可以很容易帮助你排查代码。如果问题是可重现的，这还可以让你验证得出问题是在 Astro 代码内部产生的，让你能够自信地提交一个问题报告。
+
+请注意，并非所有问题都可以在StackBlitz中复现。例如，你的问题可能依赖于特定的环境或包管理器，或者可能涉及到 HTML 流式传输，而这在 StackBlitz 中是不支持的。在这种情况下，请使用命令行工具（CLI）创建一个新的最小化的（空白的）Astro项目来复现问题，并将其上传到 GitHub 代码库中。提供一个指向你最小化复现的 GitHub 代码库的链接，而不是分享一个 StackBlitz 的 URL。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:236]:
```suggestion
请注意，并非所有问题都可以在 StackBlitz 中复现。例如，你的问题可能依赖于特定的环境或包管理器，或者可能涉及到 HTML 流式传输，而这在 StackBlitz 中是不支持的。在这种情况下，请使用命令行工具（CLI）创建一个新的最小化的（空白的）Astro 项目来复现问题，并将其上传到 GitHub 代码库中。提供一个指向你最小化复现的 GitHub 代码库的链接，而不是分享一个 StackBlitz 的 URL。
```

File: src/content/docs/zh-cn/guides/troubleshooting.mdx [hunk 4]
```diff
@@ -200,6 +221,34 @@ const answer = sum(2, 4);
...
+
+当你的空项目设置完成后，按照以下步骤复现问题，包括添加包、更改配置和编写代码。
+
+你应该只编写最少量的必要代码来复现问题。不要复制你现有项目的其他元素，并且删除所有与问题无关的代码。
+
+### 创建一个问题
+
+如果你的问题可以重现，那么现在是时候创建一个问题并提交一个 bug 报告了！
+
+前往 GitHub上 相应的 Astro 代码库去创建一个新的问题。大多数代码库都有一个问题模板，会询问问题或要求提供信息以便提交报告。遵循这些模板很重要，因为如果你没有提供我们需要的信息，我们就不得不向你询问... 这样会导致你的问题得不到及时处理！
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/troubleshooting.mdx:248]:
```suggestion
前往 GitHub上 相应的 Astro 代码库去创建一个新的问题。大多数代码库都有一个问题模板，会询问问题或要求提供信息以便提交报告。遵循这些模板很重要，因为如果你没有提供我们需要的信息，我们就不得不向你询问……这样会导致你的问题得不到及时处理！
```

**Review by 100gle — CHANGES_REQUESTED** [review]

**Comment by 100gle** [comment]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**